sky/serve/replica_managers.py

```diff
@@ -438,12 +446,13 @@ def get_ready_replica_urls(self) -> List[str]:
         """Get all ready replica's IP addresses."""
         raise NotImplementedError
 
-    def scale_up(self, n: int) -> None:
-        """Scale up the service by n replicas."""
+    def scale_up(self,
+                 resources_override: Optional[Dict[str, Any]] = None) -> None:
+        """Scale up the service by 1 replica with resources_override."""
```

Comment:
nit: add explanation an example for `resources_override` (sth like `resources_override is of the same format with resources section in skypilot task yaml`)